lib/src/authentication/widgets/input_email.dart

```diff
@@ -18,7 +18,7 @@ class InputEmail extends StatefulWidget {
       this.isRequired = true,
       this.autofocus = false,
       this.enabled = true,
-      this.textInputAction = TextInputAction.continueAction,
+      this.textInputAction = TextInputAction.none,
```

Comment:
Why change it to "none"? 🤔

Reply:
Pas supporté sur Android

Reply:
J'avais pas vu ton message 😅 dans ce cas au lieu de `none` est ce que tu peux mettre `next` qui est plus approprié en plus que `continueAction`